environment/src/main/java/com/sequenceiq/environment/telemetry/service/AccountTelemetryService.java

```diff
@@ -28,28 +38,25 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AccountTelemetryService.class);
 
-    private static final String EMAIL_PATTERN = "\\b([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-\\._]" +
-            "*[A-Za-z0-9])@(([A-Za-z0-9]|[A-Za-z][A-Za-z0-9\\-]*[A-Za-z0-9])\\.)+([A-Za-z0-9]" +
-            "|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])\\b";
-
-    private static final String EMAIL_REPLACEMENT = "email@redacted.host";
-
-    private static final String CREDIT_CARD_PATTERN = "\\d{4}[^\\w]\\d{4}[^\\w]\\d{4}[^\\w]\\d{4}";
-
-    private static final String CREDIT_CARD_REPLACEMENT = "XXXX-XXXX-XXXX-XXXX";
-
-    private static final String SSN_PATTERN = "\\d{3}[^\\w]\\d{2}[^\\w]\\d{4}";
-
-    private static final String SSN_REPLACEMENT = "XXX-XX-XXXX";
-
-    private static final String FREEIPA_PWD_PATTERN = "FPW\\:\\s+[\\w|\\W].*";
+    private final AccountTelemetryRepository accountTelemetryRepository;
 
-    private static final String FREEIPA_PWD_REPLACEMENT = "FPW: [REDACTED]";
+    private final String rulesFileLocation;
 
-    private final AccountTelemetryRepository accountTelemetryRepository;
+    private List<AnonymizationRule> defaultRules = new ArrayList<>();
 
-    public AccountTelemetryService(AccountTelemetryRepository accountTelemetryRepository) {
+    public AccountTelemetryService(AccountTelemetryRepository accountTelemetryRepository,
+            @Value("${environment.telemetry.default.rules.file:telemetry/default-anonymization-rules.yaml") String rulesFileLocation) {
         this.accountTelemetryRepository = accountTelemetryRepository;
+        this.rulesFileLocation = rulesFileLocation;
+    }
+
+    @PostConstruct
+    public void init() {
+        try {
+            defaultRules = loadAnonymizationRules();
+        } catch (IOException e) {
+            LOGGER.error("Error during loading default anonymization rules.", e);
+        }
```

Comment:
I think we won't notice this issue when something is wrong if we let the app start